proto/regen/ecocredit/v1beta1/marketplace/state.proto

```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
+
+package regen.ecocredit.v1beta1.marketplace;
+
+import "cosmos/orm/v1alpha1/orm.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// SellOrder represents the information for a sell order.
+message SellOrder {
+
+    option (cosmos.orm.v1alpha1.table) = {
+        id: 1,
+        primary_key: {fields: "order_id"}
+        index: {id: 1 fields: "batch_denom"}
+        index: {id: 2 fields: "owner"}
+    };
```

Comment:
Following #671, we need to include the `expiration` message field and add an expiration index.

Reply:
https://github.com/regen-network/regen-ledger/pull/700/commits/22491e9d5d575fadaa875d201750bf95933ea5c9

Reply:
Looks like the expiration index is still missing..

Reply:
oh whoops, think i missed the index part and only added it as a field 😅 its in there now!